src/views/Sale/components/Timer/index.tsx

```diff
@@ -12,6 +19,14 @@ import { isSaleOpen, isSaleUpcoming } from 'src/mesa/sale'
 
 // Utils
 import { convertUtcTimestampToLocal } from 'src/utils/date'
+import { timezoneAbbreviation } from 'src/utils/timezone'
+
+dayjs.extend(advanced)
+dayjs.extend(utc)
+dayjs.extend(timezone)
+dayjs.extend(advanced)
+dayjs.extend(relativeTime)
+dayjs.extend(durationTime)
```

Comment:
These are set up/configuration calls. Let's move these to `src/App.ts` or `src/index.ts`